### PR TITLE
Empty arrays are truthy in JS

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: src/components/Eligibility.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ class Eligibility extends React.Component {
         var eligibleMarkup;
         var ineligibleMarkup;
 
-        if (eligibilityInfo || catchment) {
+        if (eligibilityInfo.length || catchment.length) {
```
Comment (danni) [inline src/components/Eligibility.js:34]: `_.isEmpty` is more lucid IMO

**Comment by danni** [comment]
Tests?

**Comment by danni** [comment]
++

However I think we should break the service details page out into a number of features.

**Comment by DanielHeath** [comment]
It's definitely getting overly long.

**Pull request merged**